cheddar/cheddar-server/src/main/java/com/clicktravel/cheddar/server/application/logging/LoggingAspect.java

```diff
@@ -60,13 +60,13 @@ private Object proceedAndLog(final ProceedingJoinPoint point) throws Throwable {
         final String securityContextDetail = securityContextDetail();
         final long startTime = System.currentTimeMillis();
         try {
-            logger.debug("Entering method call; call:[{}] {}", methodCallDetail, securityContextDetail);
+            logger.debug("Enter;[{}] {}", methodCallDetail, securityContextDetail);
             final Object result = point.proceed();
-            logger.debug("Method call returned; call:[{}] return:[{}] timeMillis:[{}] {}", methodCallDetail, result,
-                    System.currentTimeMillis() - startTime, securityContextDetail);
+            logger.debug("Return;[{}] ms:[{}] {}", result, System.currentTimeMillis() - startTime,
+                    securityContextDetail);
             return result;
         } catch (final Throwable e) {
-            logger.debug("Method call exception; call:[{}] exception:[{}] timeMillis:[{}] {}", methodCallDetail, e,
+            logger.debug("Exception;[{}] exception:[{}] ms:[{}] {}", methodCallDetail, e,
```

Comment:
As above, the security context detail can be removed from here too

Reply:
Ive removed that extra logging as suggested, thanks for heads up